neo4j/neo4j.yaml

```diff
@@ -0,0 +1,12 @@
+init_config:
```

Comment:
This file looks to have been committed by mistake. We should only include conf.yaml.example.